gcli/tests/cmd/upload.rs

```diff
@@ -30,9 +25,6 @@ async fn test_command_upload_works() {
         &env::wasm_bin("demo_meta.opt.wasm"),
     ])
     .expect("run command upload failed");
-    assert!(api
-        .code_storage(code_hash)
-        .await
-        .expect("get code failed")
-        .is_some());
+
+    assert!(api.code_storage(code_hash).await.is_ok());
```

Comment:
how we gonna understand now if there some api troubles or value doesn't exist in storage?